Makefile

```diff
@@ -51,8 +51,8 @@ downloads/drive:
 
 ########################################################################################################
 
-CHAT_VERSION = 2.0.1
-CHAT_LINK = "https://s3-eu-west-1.amazonaws.com/zextras-artifacts/openchat/20_Jul_2018_OP-CPB-34/openchat.tgz"
+CHAT_VERSION = 2.0.2
+CHAT_LINK = "https://files.zimbra.com/downloads//openchat.tgz"
```

Comment:
This seems like a scary source URL - no version info, could get re-used in the future.  Would prefer it to be moved somewhere where it is more obvious what it is.

Reply:
This is temporarily and the only available  package as of now. As zextras has some issue with there bucket for the time being. Since I use files.zimbra.com for the time being.

Reply:
Can't you rename the file on files.zimbra.com though???